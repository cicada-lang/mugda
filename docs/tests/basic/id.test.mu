(import "./id.mu" id idType)

id
(id idType id)
((id idType id) idType (id idType id))
