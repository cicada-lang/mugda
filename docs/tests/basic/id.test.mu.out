(lambda (A a) a)
(lambda (A a) a)
(lambda (A a) a)
